<launch>
  <!-- load params of the joint velocity controllers -->
  <rosparam command="load" file="$(find ias_mechanism_controllers)/launch/pr2_multi_vel.yaml"/>

  <!-- load the calibration parameters for the joint velocity controllers -->
  <rosparam command="load" file="$(find ias_mechanism_controllers_calibration)/launch/pr2_multi_vel_calibration_params.yaml" />

  <!-- load the velocity-resolved controllers in stopped state -->
  <node name="spawner_vel_controllers" pkg="pr2_controller_manager" type="spawner" args="--stopped l_arm_vel r_arm_vel" />

  <!-- start the test node -->
  <node name="left_arm_test" pkg="ias_mechanism_controllers_calibration" type="pr2_multi_vel_test" output="screen">
    <remap from="arm_trajectory_action" to="/l_arm_controller/joint_trajectory_action"/>
    <remap from="~multi_vel_command" to="/l_arm_vel/command" />
    <param name="~arm_action_server_namespace" value="/l_arm_controller" />
    <param name="~vel_controller_namespace" value="/l_arm_vel" />
  </node>
</launch>
